### ISSUE TITLE
Password string is not encoded as URI

### ISSUE THREAD

**Issue by System32-0101** [wxyz-abcd/node-haxball#72]
Password string is not encoded as URI so it will drop an error when you try to enter in a match that includes special characters in the password like (!,@,#,$,%,^, etc...)

Way I fixed:
In [joinRoom.js](https://github.com/wxyz-abcd/node-haxball/blob/main/examples_web/src/joinRoom.js)
Line: 34:
Fix:
```js
            "r_pass": encodeURIComponent(document.getElementById("r_pass").value.toString()),
```

In: [init.js](https://github.com/wxyz-abcd/node-haxball/blob/main/examples_web/src/init.js)
Line: 92
Fix: 
```js
                    Room.join({
          id: params.r_id,
          password: decodeURIComponent(params.r_pass),
          token: params.token,
          authObj: authObj
        }
```
This way we assign the password to the decoded URI component that we coded before.

Hope this works :)

**Comment by wxyz-abcd** [comment]
Thanks for the attention and the issue. Can you try undoing them and changing this line instead?

In: [examples_web/src/htmlQueryHelper.js](htmlQueryHelper.js) Line: 12

`
  return Object.keys(json).filter((x)=>(json[x]!=null && json[x]!="")).map((x)=>(encodeURI(x)+"="+encodeURI(json[x]))).join("&");
`

to:

`
  return Object.keys(json).filter((x)=>(json[x]!=null && json[x]!="")).map((x)=>(encodeURIComponent(x)+"="+encodeURIComponent(json[x]))).join("&");
`

**Comment by wxyz-abcd** [comment]
Fixed in v1.5.
